spec/hosts/roles/hexrot_spec.rb

```diff
@@ -32,16 +32,85 @@
           it { is_expected.to contain_class('docker') }
 
           %w[
+            anaconda
+            mate
             profile::core::common
```

Comment:
This duplicates ` include_examples 'common', os_facts: os_facts`

Reply:
removed 👍🏻